crates/test-programs/wasi-tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+Cargo.lock
```

Comment:
We currently intentionally check in the lock file, so mind removing this file?

Reply:
Of course. I thought it was an oversight. I'll remove the gitignore.